### ISSUE TITLE
Guides and examples for 2nd phase retrieval 

### ISSUE THREAD

**Issue by joemcelroy** [elastic/elasticsearch-labs#88]
Providing examples on how to re-rank using bi-encoders and cross-encoders.

**Comment by wenestam** [comment]
Great suggestion.

Similar to how embedding pipelines can be added: https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-text-emb-vector-search-example.html

or here:

https://www.elastic.co/search-labs/how-to-deploy-nlp-text-embeddings-and-vector-search

It would be cool to utilize the machine learning instance for re-ranking. Similarly, you could bring your own model via hugging face by installing it through the Eland Docker agent (as in the later link).

**Comment by serenachou** [comment]
cc @joshdevins  @leemthompo  related to our conversation around guides and documentation for reranking

**Comment by joshdevins** [comment]
Yeah we should coordinate on this. We have some plans for documentation around this as well so let's make sure they are all talking about the same things.
